app/src/main/java/com/google/samples/apps/sunflower/compose/utils/SunflowerImage.kt

```diff
@@ -59,6 +64,14 @@ fun SunflowerImage(
         contentScale = contentScale,
         alpha = alpha,
         colorFilter = colorFilter,
-        requestBuilderTransform = requestBuilderTransform
+        requestBuilderTransform = requestBuilderTransform,
+            loading = placeholder {
+                Column(modifier=modifier.fillMaxWidth(),
```

Comment:
This should be a `Box` instead

Reply:
Actually I am not that experience in Compose but when I use Box with the fillMaxWidth like this
```
loading = placeholder {
            Box(modifier.fillMaxWidth()) {
                   CircularProgressIndicator(modifier.align(Alignment.Center))
             }
}
```
I got an ugly loader fill all the widget 
[device-2023-07-01-083420.webm](https://github.com/android/sunflower/assets/1716129/ac923177-e2fb-4710-acc3-fbb39cd382e9)

Reply:
Now it is done